assets/scss/core/_variables.scss

```diff
@@ -25,10 +20,24 @@ $dp-font-color-bright: #edede6 !default;
 $dp-font-color-dark: $dp-font-color !default;
 $dp-font-color-white: #fff !default;
 
+////
+/// Font Size settings
+///
+$dp-sizing--lvl10: 60px;
```

Comment:
I'm just asking here but: Don't we already have numbers like this in others variables? why we dont just use them here at least.